packages/inter-protocol/test/smartWallet/boot-psm.js

```diff
@@ -177,6 +178,49 @@ export const ParametersShape = M.splitRecord(
   },
 );
 
+/**
+ * @deprecated use publishAgoricNamesToChainStorage
+ * @param {BootstrapPowers} powers
+ * @param {{
+ *   options?: {
+ *     agoricNamesOptions?: {
+ *       topLevel?: string[];
+ *     };
+ *   };
+ * }} config
+ */
+export const legacyPublishAgoricNames = async (
+  { consume: { agoricNamesAdmin, board, chainStorage: rootP } },
+  { options: { agoricNamesOptions } = {} } = {},
+) => {
+  const root = await rootP;
+  assert(root, 'legacyPublishAgoricNames requires chainStorage');
+  const nameStorage = E(root).makeChildNode('agoricNames');
+  const marshaller = E(board).getPublishingMarshaller();
+
+  // XXX will fail upon restart, but so would the makeStoredPublishKit this is replacing
+  // Since we expect the bootstrap vat to be replaced instead of upgraded this should be
+  // fine. See {@link ./README.md bootstrap documentation} for details.
```

Comment:
Where is that link supposed to go? I don't see a README in this dir.

Possible alternative link:

 - https://github.com/Agoric/agoric-sdk/issues/8849